napari_aicssegmentation/view/workflow_steps_view.py

```diff
@@ -200,10 +210,21 @@ def _btn_close_keep_clicked(self, checked: bool):
         self._controller.close_workflow()
 
     def _btn_run_all_clicked(self, checked: bool):
-        workflow_step_widgets: List[WorkflowStepWidget] = self.findChildren(WorkflowStepWidget)
-        all_parameter_inputs = [w.get_parameter_inputs() for w in workflow_step_widgets]
+        all_parameter_inputs = [w.get_parameter_inputs() for w in self._get_workflow_step_widgets()]
         self._controller.run_all(all_parameter_inputs)
 
     def _btn_run_all_cancel_clicked(self, checked: bool):
         self.btn_run_all.setText("Canceling...")
         self._controller.cancel_run_all()
+
+    def _btn_save_workflow_clicked(self, checked: bool):
+        file_path, _ = QFileDialog.getSaveFileName(
```

Comment:
[nit] I had to look up what this `_` variable was -- if it's a string containing filters, maybe call it `filters` instead or if we're never going to use `filters` then not destructure the tuple that's returned? OR at the very least add a comment what `_` is :)

Reply:
@schoinh in Python, by convention, the single underscore is used to unpack values you want to ignore when using tuples or loops. See the first part of https://betterprogramming.pub/the-special-meaning-of-underscores-in-python-9ceaaeb41007

Reply:
Ohh! Learned something new, thanks!